src/lib/skilling/skills/mining.ts

```diff
@@ -259,6 +259,20 @@ const MotherlodeMine: Ore = {
 	petChance: 247_200
 };
 
+// Uses determineMiningTime function, therefore Ore object and id -1
+const CamdozaalMine: Ore = {
+	level: 14,
+	xp: -1,
```

Comment:
You could declare this xp as 16. And double it later on to get the xp value for barronite deposits.

Reply:
good idea, fixed